### ISSUE TITLE
Windows x86_64 support

### ISSUE THREAD

**Issue by slimsag** [hexops/mach#5]

**Comment by wilsonk** [comment]
This small patch will get things to build on Win10 using MSYS2 or git-bash. All 32 tests pass with this patch.

```
diff --git a/glfw/build.zig b/glfw/build.zig
index a0ca24b..1169031 100644
--- a/glfw/build.zig
+++ b/glfw/build.zig
@@ -71,7 +71,7 @@ pub fn link(b: *Builder, step: *std.build.LibExeObjStep, options: Options) void
                 var abs_path = std.fs.path.join(&arena.allocator, &.{ thisDir(), path }) catch unreachable;
                 sources.append(abs_path) catch unreachable;
             }
-            lib.addCSourceFiles(sources.items, &.{});
+            lib.addCSourceFiles(sources.items, &.{"-D_GLFW_WIN32"});
         },
         .macos => {
             includeSdkMacOS(b, lib);
@@ -175,6 +175,7 @@ fn linkGLFW(b: *Builder, step: *std.build.LibExeObjStep, options: Options) void
     switch (target.os.tag) {
         .windows => {
             // TODO(slimsag): create sdk-windows
+            step.linkSystemLibrary("gdi32");
             step.linkSystemLibrary("c");
         },
         .macos => {
```

**Comment by slimsag** [comment]
Fixed by #14 - Windows cross compilation from all OS is now verified by CI as well. Our platform support is getting there... 🏃 

<img width="923" alt="image" src="https://user-images.githubusercontent.com/3173176/126941893-be9c339c-0192-4d7e-9a5e-a8a146d34ee6.png">

**Comment by wilsonk** [comment]
Nice :)
